.github/workflows/integrations.yml

```diff
@@ -0,0 +1,53 @@
+
+name: RMT Integration tests
+
+on: [pull_request]
+
+jobs:
+  test:
+    env:
+      DANGER_GITHUB_API_TOKEN: ${ secrets.GH_TOKEN }
+    services:
+      mysql:
+        image: mysql:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_DATABASE: rmt_test
+          MYSQL_USER: rmt
+          MYSQL_PASSWORD: rmt
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.5.8', '2.6.6', '2.7.2']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+        bundler: 1.17
+    - name: Gem cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ matrix.ruby-version }}-gems-${{ hashFiles('Gemfile') }}
+        restore-keys: |
+          ${{ matrix.ruby-version }}-gems-
+    - name: Pre-run inline scripts
+      run: |
+        ruby -e "require 'yaml'; puts({'database_test'=>{'host'=>'0.0.0.0','username'=>'rmt','password'=>'rmt','database'=>'rmt_test','adapter'=>'mysql2','encoding'=>'utf8','timeout'=>5000,'pool'=>5}}.to_yaml)" > config/rmt.local.yml
```

Comment:
Now, that's something from RMT's code that I don't fully grasp. First of all, why isn't there a `config/database.yml` file as any other Rails application? Second of all, don't we have this exact configuration in `config/rmt.yml`? Why don't we pick our default settings?

Reply:
`lib/rmt/config.rb` explains the process. If you open `application.rb` file, you can see how the database config is created.